From ed1c0b474c3b3e9c588122d84da10c885543365d Mon Sep 17 00:00:00 2001
From: Michael Sarahan <msarahan@gmail.com>
Date: Mon, 12 Nov 2018 09:14:24 -0600
Subject: [PATCH] remove win dll download

---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 7cd99bd..b1882a8 100755
--- a/setup.py
+++ b/setup.py
@@ -311,7 +311,7 @@ if WITH_SETUPTOOLS:
             if IS_WINDOWS_PLATFORM:
                 # Download the required DLLs
                 self.distribution.salt_download_windows_dlls = True
-                self.run_command('download-windows-dlls')
+                # self.run_command('download-windows-dlls')
                 self.distribution.salt_download_windows_dlls = None
 
             if self.write_salt_version is True:
@@ -671,7 +671,7 @@ class Install(install):
         if IS_WINDOWS_PLATFORM:
             # Download the required DLLs
             self.distribution.salt_download_windows_dlls = True
-            self.run_command('download-windows-dlls')
+            # self.run_command('download-windows-dlls')
             self.distribution.salt_download_windows_dlls = None
         # Run install.run
         install.run(self)
-- 
2.19.1

